apps/greencheck/migrations/0022_delete_dailystat_delete_greencheckstats_and_more.py

```diff
@@ -0,0 +1,93 @@
+# Generated by Django 5.0.9 on 2025-06-23 06:44
+
+import django_mysql.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("greencheck", "0021_add_created_to_greendomains"),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
```

Comment:
*if* we want to keep these tables around for any reason, we should wrap these in a SeparateDatabaseFromState instance to update the schema without deleting the data. To discuss with @mrchrisadams

Reply:
We're happy these can go - we've got backups!